extensions/scarb-snforge-test-collector/src/main.rs

```diff
@@ -22,12 +23,19 @@ mod metadata;
 struct Args {
     #[command(flatten)]
     packages_filter: PackagesFilter,
+
+    #[command(flatten)]
+    pub features: FeaturesSpec,
```

Comment:
When I think about this now, this env should be set by Foundry and just inherited downstream, no need for cli parser.

Reply:
Hmmm okay I guess